docs/user-guides/workspace-access/port-forwarding.md

```diff
@@ -6,17 +6,19 @@ Port forwarding lets developers securely access processes on their Coder
 workspace from a local machine. A common use case is testing web applications in
 a browser.
 
-There are three ways to forward ports in Coder:
+There are multiple ways to forward ports in Coder:
 
-- The `coder port-forward` command
-- Dashboard
-- SSH
+| Method                                                          | Details                                                                                                                     |
+|:----------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------|
+| [Coder Desktop](../desktop/index.md)                            | Uses a VPN tunnel to your workspace and provides access to all running ports.                                               |
+| [`coder port-forward` command](#the-coder-port-forward-command) | Can be used to forward TCP or UDP ports from the remote workspace so they can be accessed locally.                          |
+| [Dashboard](#dashboard)                                         | Proxies traffic through the Coder control plane versus peer-to-peer which is possible with the Coder CLI and Coder Desktop. |
+| [SSH](#ssh)                                                     | Double-encrypts traffic with both Wireguard and SSH.                                                                        |
```

Comment:
```suggestion
| [SSH](#ssh)                                                     | Forwards ports over an SSH connection.                                                                                      |
```

Reply:
What do you think about focusing on how it works here instead of how its different?